### PR TITLE
Patch 4

### LINKED ISSUE #48: Windows PowerShell unable to parse ANSI color codes
Oddly enough, a new shell session completely nixed my ability to ANSI colored output on windows. I am not sure how it was ever displaying colors since PowerShell is not able to parse ANSI color codes.

image

If anyone else can confirm this issue then we can begin to investigate as I am not sure if it is environment specific to my machine.

### PR DESCRIPTION
**Pull request by JasonBristol**
1. Fixed #48 & #49 
2. Renamed helper functions
3. Changed logging colors
4. Disabled windows logging colors (unsupported)

### CODE REVIEW

File: wrapper/utils/log.py [hunk 1]
```diff
@@ -158,24 +158,25 @@ def format(self, record):
...
-            error_style = termcolors.make_style(fg="red", opts=("bold",))
-            msg = error_style(msg)
-        elif record.levelno == logging.CRITICAL:
-            crit_style = termcolors.make_style(fg="black", bg="red", opts=("bold",))
-            msg = crit_style(msg)
-        elif record.levelno == logging.TRACE:
-            trace_style  = termcolors.make_style(fg="black", bg="white")
-            msg = trace_style(msg)
+        if os.name =="posix": # Only style on linux since windows doesn't support ANSI
+            if record.levelno == logging.INFO:
```
Comment (suresttexas00) [inline wrapper/utils/log.py]: would it hurt to allow this for all OS's?  maybe there is a fluke that it sometimes works and sometimes not on Windows.  Would it cause an error to allow the setting to continue for all OS's?  did you try this on a default cmd.exe?

File: wrapper/core/commands.py [hunk 2]
```diff
@@ -4,8 +4,7 @@
 import ast
 import random
 
-from utils.helpers import args as get_args
-from utils.helpers import argsAfter as get_argsAfter
+from utils.helpers import get_args, get_argsAfter
```
Comment (suresttexas00) [inline wrapper/core/commands.py]: Yea, the name collisions were why i originally changed the imports to rename the args (in particular) from helpers to get_args.

File: wrapper/utils/log.py (hunk 1 above)
Comment (JasonBristol) [inline wrapper/utils/log.py]: We could enable this for Mac as well, but for Windows you would need a
different shell entirely like cygwin, ANSI support disappeared after win 98
so even cmd.exe won't work

Comment (suresttexas00) [inline wrapper/utils/log.py]: what will this do in windows?  Does it cause an error or garble the screen with the graphics codes or does it ignore them?

**Pull request merged**